OpenConsole.sln

```diff
@@ -1863,6 +2055,7 @@ Global
 		{024052DE-83FB-4653-AEA4-90790D29D5BD}.AuditMode|x86.ActiveCfg = AuditMode|Win32
 		{024052DE-83FB-4653-AEA4-90790D29D5BD}.AuditMode|x86.Build.0 = AuditMode|Win32
 		{024052DE-83FB-4653-AEA4-90790D29D5BD}.Debug|Any CPU.ActiveCfg = Debug|Win32
+		{024052DE-83FB-4653-AEA4-90790D29D5BD}.Debug|ARM.ActiveCfg = Debug|Win32

```

Comment:
why are we adding an `ARM` solution configuration to every project?

Reply:
I... uh... don't know. I can take that out.